spec.bs

```diff
@@ -5905,7 +6104,13 @@ An <dfn>auction config</dfn> is a [=struct=] with the following [=struct/items=]
     Each buyer's real time reporting type. Currently the only supported type is
     "default-local-reporting" indicating local differential privacy. All buyers in the map opted in
     to receive real time reports.
-
+  : <dfn>server response</dfn>
+  :: Null or a {{Promise}} or a {{Uint8Array}} containing an encrypted response from the trusted auction server.
+  : <dfn>server response id</dfn>
+  :: Null or a [=version 4 UUID=]
+    A UUID used to match a request from
+    {{Window/navigator}}.{{Navigator/getInterestGroupAdAuctionData()}} to the
+    encrypted response stored in [=auction config=]'s [=auction config/server response=] field.

```

Comment:
Add a sentence about "it must be null when server response is null, and not null when server response is not null" below

Reply:
@qingxinwu WDYT of:
```suggestion
    encrypted response stored in [=auction config=]'s [=auction config/server response=] field.
    Must be null when [=auction config=]'s [=auction config/server response=] is null,
    and non-null otherwise.
```